spec/system/consumer/registration_spec.rb

```diff
@@ -70,6 +70,10 @@
       expect(page).to have_content 'Last step to add My Awesome Enterprise!'
 
       # Choosing a type
+      click_button "Create Profile"
+      expect(page).to have_content("Please choose one. Are you are producer?")
+      expect(page).to have_button "Create Profile", disabled: false
+
       click_link "producer-panel"
       expect(page).to have_selector '#producer-panel.selected'
 
```

Comment:
We should still assure that the button now becomes active, after clicking the producer panel - right?

```suggestion
      expect(page).to have_button "Create Profile", disabled: false
```

Reply:
Correct.
But I assumed the next line `click_button "Create Profile"` would only work if its enabled.
Please let me know if my understanding is incorrect.